.github/workflows/dotnet-format-md.yml

```diff
@@ -0,0 +1,14 @@
+name: dotnet format
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/.github/workflows/dotnet-format-md.yml please add similar comments here too to make it easy for future readers to understand why this is needed.